### PR TITLE
Emily Lab 4.2 Issues

### LINKED ISSUE #286: Lab 4.2 Doubly Linked List Content Stlying
71.md:
* cut some information, there is too much implementation given away

711.md:
*  split the card and code into steps
* again does order matter?

### PR DESCRIPTION
**Pull request by emily-gui**
Closes #286

### CODE REVIEW

**Comment by emily-gui** [comment]
Closes #285

**Comment by emily-gui** [comment]
Closes #283

File: Module4_Labs/Lab2_Doubly_Linked_List/Cards/5.md [hunk 1]
```diff
@@ -1,47 +1,98 @@
...
 
-**Inserting Items in Empty List**
+- Define a function called **popFront()** which does not take in any parameter.
+- If the list is empty, return.
+- Otherwise, delete the head node.
 
-- Write a function called **push()** that takes in a element as the parameter, and check whether the list is empty.
-- If this is an empty list, insert the new node. Otherwise, you will continue to the next step below, "inserting items at the start."
+```python
+def popFront(self):
```
Comment (mxthu313) [inline Module4_Labs/Lab2_Doubly_Linked_List/Cards/5.md]: why are there code blocks in the hard card?

File: Module4_Labs/Lab2_Doubly_Linked_List/Cards/5.md [hunk 2]
```diff
@@ -1,98 +1,47 @@
...
-	if self.head == None:
-  		return
+**Inserting Item After Another Item**
 
-	else:
-  		# 2. If there ISN'T only one node
-    	if self.head != None:
-    		self.tail = self.tail.prev
-    		self.tail.next = None
+- Define a function called **insertAfter()** which will take two parameters: the value of new node and the node to be inserted after.
```
Comment (mxthu313) [inline Module4_Labs/Lab2_Doubly_Linked_List/Cards/5.md]: fix indentation

File: Module4_Labs/Lab2_Doubly_Linked_List/Cards/5.md [hunk 3]
```diff
@@ -1,98 +1,47 @@
...
-            
-            if curr == None:
-            	print("Target not found.")
-                return
-
-    # Decrement size
-    self.size -= 1
-```
\ No newline at end of file
+- Define a function called **insertBefore()** which will take two parameters for the value of new node and the node to be inserted after.
```
Comment (mxthu313) [inline Module4_Labs/Lab2_Doubly_Linked_List/Cards/5.md]: fix indentation

**Pull request merged**